internal/graph/check.go

```diff
@@ -63,6 +63,7 @@ func WithUsersetBatchSize(usersetBatchSize uint32) LocalCheckerOption {
 }
 
 // WithMaxConcurrentReads see server.WithMaxConcurrentReadsForCheck.
+// TODO remove - unused.
```

Comment:
want to lift this up as an issue?

Reply:
https://github.com/openfga/openfga/issues/1920